malduck/crypto/components/pyserpent.py

```diff
@@ -151,7 +136,7 @@ def byteswap32(x):
 def set_key(l_key: List[int], key: List[int], key_len: int) -> None:
     key_len *= 8
     if key_len > 256:
-        return False
```

Comment:
:concerned:

Reply:
It shouldn't change the behaviour, right?

Reply:
No,  I was concerned about this unexpected `return False` here :D